File: w41.c
#include <stdio.h>

int isPrime(int n) {
    if (n <= 1) return 0;  // Numbers less than or equal to 1 are not prime.
    
    for (int i = 2; i < n; i++) {
        if (n % i == 0) return 0;  // If divisible by any number other than 1 and itself, not prime.
    }
    
    return 1;  // Number is prime.
}

int main() {

File: w41.py
def is_prime(n):
    if n <= 1:
        return False  # Numbers less than or equal to 1 are not prime.
    
    for i in range(2, n):
        if n % i == 0:
            return False  # If divisible by any number other than 1 and itself, not prime.
    
    return True  # Number is prime.

number = int(input("Enter a number: "))

if is_prime(number):
    print(f"{number} is a prime number.")
else:
    print(f"{number} is not a prime number.")
